src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/TBD",
+        "display": "Doe, John TBD"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for ecog12]"
+                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://snomed.info/sct",
+                            "code": "719091000000102",
+                            "display": "Questionnaire"
+                        }
+                    ],
+                    "text": "Everyday Cognition (ECOG) 12 (short form) questionnaire"
+                },
+                "status": "active",
+                "scheduledTiming": {
+                    "repeat": {
+                        "frequency": 1,
+                        "period": 1,
+                        "periodUnit": "d"
+                    }
+                }
+            }
+        },
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for Mini-Cog]"
```

Comment:
@achen2401 @pbugni  not sure what to do about an ID here... I assume that it wouldn't be consistent across HAPI instances?

Reply:
@mcjustin right now the app loads the questionnaire from the SoF host by searching for it by [name](https://github.com/uwcirg/asbi-screening-app/blob/master/src/util/screening-selector.js#L43)  Will that change? (Search by id instead?)

Reply:
Resolved this w/ https://github.com/uwcirg/asbi-screening-app/pull/26/commits/a8d84950a5e803e381c11bf2f8e348ab4aeada0d